app/api/helpers/system_notifications.py

```diff
@@ -392,7 +392,7 @@ def get_invite_papers_notification_actions(cfs_link, submit_link):
     },
     EVENT_ROLE: {
         'title': u'Invitation to be {role_name} at {event_name}',
-        'message': u"You've been invited to be a <strong>{role_name}</strong>" +
+        'message': u"You've been invited to be one of the <strong>{role_name}</strong>s" +
```

Comment:
Just replace `a` with `an` as mentioned in the issue

Reply:
@shreyanshdwivedi It will require a condition block too then, Its a nice makeover done by him 😄

Reply:
Then the `s` must be in `strong` block itself